source/configure/calls-deployment.rst

```diff
@@ -181,7 +181,10 @@ Both the plugin and the external ``rtcd`` service expose some Prometheus metrics
 Calls plugin metrics
 ^^^^^^^^^^^^^^^^^^^^
 
-Metrics for the calls plugin are exposed through the public ``/plugins/com.mattermost.calls/metrics`` API endpoint.
+Metrics for the calls plugin are exposed through the ``/plugins/com.mattermost.calls/metrics`` subpath under the existing Mattermost server metrics endpoint.
+
+.. note::
+   On Mattermost versions prior to v9.5, plugin metrics were exposed through the public ``/plugins/com.mattermost.calls/metrics`` API endpoint.
```

Comment:
I think this might be a little confusing, because it's the same as the above url...

Reply:
@cpoile Would you suggest we explain further the difference in HTTP listeners or rather remove the note entirely and consider the recent versions only? 

I'd probably lean on the latter as it simplifies, especially given the new endpoint has been there since the latest ESR (February).

Reply:
Sorry, I think I missed this ping.